src/AvaloniaEdit/Editing/TextArea.cs

```diff
@@ -745,12 +756,14 @@ protected override void OnGotFocus(GotFocusEventArgs e)
             // First activate IME, then show caret
```

Comment:
Does the order matters? Old comments said:
First activate IME, then show caret

Reply:
This is handled by Avalonia so a control that wants to interact with IME just has to implement `ITextInputMethodClient`

Reply:
Will change order just for consistency